adal/src/androidTest/java/com/microsoft/aad/adal/AcquireTokenRequestTest.java

```diff
@@ -406,20 +396,30 @@ public void testLocalSilentFailedBrokerSilentReturnErrorCannotTryWithInteractive
         cacheStore.removeAll();
     }
 
-    private ITokenCacheStore getTokenCache(final Date expiresOn, final boolean storeMRRT, final boolean storeFRT) {
+    private ITokenCacheStore getTokenCache(final int expiresOn, final boolean storeMRRT, final boolean storeFRT, final Integer extendedExpiresOn) {
```

Comment:
If you insist on using object here, I'll leave it up to you and others. But here is the general rule 1) always prefer int for performance benefit 2) if you have to use object in a oo manner, easily boxing and unboxing data, use the object. To be honest, here I didn't see why we have to use it